# DP: Don't turn on DWARF 5 by default, where GCC 11 is not the default.

--- a/src/gcc/common.opt
+++ b/src/gcc/common.opt
@@ -3175,7 +3175,7 @@ Common Driver JoinedOrMissing Negative(g
 Generate debug information in default version of DWARF format.
 
 gdwarf-
-Common Driver Joined UInteger Var(dwarf_version) Init(5) Negative(gstabs)
+Common Driver Joined UInteger Var(dwarf_version) Init(4) Negative(gstabs)
 Generate debug information in DWARF v2 (or later) format.
 
 gdwarf32
